src/app/api/faucet/verify/route.ts

```diff
@@ -0,0 +1,60 @@
+import { NextResponse } from 'next/server';
+import { initializeClients, publicClient } from '../../../../lib/client';
+import { abi, address } from '../../../../lib/constants';
+
+async function verify(userAddress: string) {
+    try {
+        initializeClients();
```

Comment:
you can just say const {publicClient} = initializeClients();

That way you dont have to import is separately, or add a check for !publicClient